### DISCUSSION TITLE
Quesiton regarding Backend migration

### DISCUSSION THREAD

**Discussion by StackSurfer** [Bootstrap-Academy/Bootstrap-Academy#98] [Q&A] [answered]
I was about to read some API documentation when I discovered that the old ones are no longer functional. I understand that there's a planned migration for the Backend. Is there a new method to access the updated routes and API documentation?

**Comment by Defelo** [comment, marked as answer]
Already answered in https://github.com/Bootstrap-Academy/frontend/pull/88#issuecomment-1871222232, but for the sake of completeness: The api docs are now available at https://api.test.bootstrap.academy/. This is also documented in the [Bootstrap-Academy readme](https://github.com/Bootstrap-Academy/Bootstrap-Academy?tab=readme-ov-file#test-instance).
